src/proto/mgmt/mgmt.proto

```diff
@@ -29,6 +29,27 @@ import "srv.proto";
 import "storage_query.proto"; // storage query req/resp BIO data and SMD device list
 import "acl.proto"; // ACL-related requests
 
+// Generic request indicating which ranks to operate on.
+// Used in gRPC fanout to operate on hosts with multiple ranks.
+message RanksReq {
+	repeated uint32 ranks = 1;
+	float timeout = 2;
+}
+
+// Generic response containing DER result from multiple ranks.
+// Used in gRPC fanout to operate on hosts with multiple ranks.
+message RanksResp {
+	// Identical to ctl.system.RankResult proto message.
+	message RankResult {
+		uint32 rank = 1;
+		string action = 2;
+		bool errored = 3;
+		string msg = 4;
+		uint32 state = 5;
+	}
+	repeated RankResult results = 1;
+}
```

Comment:
Could this rank stuff be separated out into its own proto file since it is shared by mgmt and ctl?

Reply:
to be addressed in https://jira.hpdd.intel.com/browse/DAOS-4033